tools/templates/aten_test_batch_norm_no_training_module.tmpl

```diff
@@ -0,0 +1,9 @@
+class AtenModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input, weight, bias, running_mean, running_var, momentum, eps):
+        # Make sure the var >= 0
+        running_var = running_var**2
```

Comment:
Can you please clarify why this is needed?

Reply:
It's specific to autogen tests, because it always generate random tensors with https://github.com/tenstorrent/pytorch2.0_ttnn/blob/2d54faa2510e0bfb5ec7ce16176c40bd9581cec7/tests/utils.py#L539 which by default will generate tensors with negative values. So this line is just to make sure it is positive.

Now I realized that this part will be compiled into TTNN, which includes extra ops under the test. And we actually can customize the generated input tensors for ops here:

https://github.com/tenstorrent/pytorch2.0_ttnn/blob/2d54faa2510e0bfb5ec7ce16176c40bd9581cec7/tests/utils.py#L412-L426

I will change to use the customization list above

Reply:
Moved the special input generation rule to `tests/utils.py`. Removed the `running_var = running_var**2` from tests